Could not resolve all artifacts for configuration ':flutter_logs:classpath'.

Hi, 
I can not seem to build Version 2.1.7

Flutter 3.3.6

`kotlin_version = '1.5.31'`

```
 dependencies {
        classpath 'com.android.tools.build:gradle:7.1.1'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }
```

`distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip`
  
Log excerpt (github action): 

```
Running Gradle task 'assembleDebug'...                          
FAILURE: Build failed with an exception.
* What went wrong:
A problem occurred configuring project ':flutter_logs'.
> Could not resolve all artifacts for configuration ':flutter_logs:classpath'.
   > Could not resolve com.android.tools.analytics-library:crash:27.1.3.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3
      > Could not resolve com.android.tools.analytics-library:crash:27.1.3.
         > Could not get resource 'https://jcenter.bintray.com/com/android/tools/analytics-library/crash/27.1.3/crash-27.1.3.pom'.
            > Could not GET 'https://jcenter.bintray.com/com/android/tools/analytics-library/crash/27.1.3/crash-27.1.3.pom'.
               > Read timed out
   > Could not resolve com.android.tools.build:bundletool:0.14.0.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3
      > Could not resolve com.android.tools.build:bundletool:0.14.0.
         > Could not get resource 'https://jcenter.bintray.com/com/android/tools/build/bundletool/0.14.0/bundletool-0.14.0.pom'.
            > Could not HEAD 'https://jcenter.bintray.com/com/android/tools/build/bundletool/0.14.0/bundletool-0.14.0.pom'.
               > Read timed out
   > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin-api:1.5.20.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin-model:1.5.20.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-util-klib:1.5.20.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-klib-commonizer-api:1.5.20.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-tooling-metadata:1.5.20.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-project-model:1.5.20.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-compiler-embeddable:1.5.20.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-annotation-processing-gradle:1.5.20.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-android-extensions:1.5.20.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-compiler-runner:1.5.20.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-scripting-compiler-embeddable:1.5.20.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-scripting-compiler-impl-embeddable:1.5.20.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20
      > Skipped due to earlier error
   > Could not resolve org.ow2.asm:asm-tree:7.0.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > org.ow2.asm:asm-analysis:7.0
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > org.ow2.asm:asm-commons:7.0
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > org.ow2.asm:asm-util:7.0
      > Skipped due to earlier error
   > Could not resolve it.unimi.dsi:fastutil:7.2.0.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-reflect:1.3.72.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > com.android.tools.lint:lint-gradle-api:27.1.3
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdk-common:27.1.3
      > Skipped due to earlier error
   > Could not resolve org.glassfish.jaxb:jaxb-runtime:2.3.1.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > androidx.databinding:databinding-compiler-common:4.1.3
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdklib:27.1.3 > com.android.tools:repository:27.1.3
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-stdlib:1.3.72.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.72
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.72.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.72
      > Skipped due to earlier error
   > Could not resolve net.sf.proguard:proguard-base:6.0.3.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > net.sf.proguard:proguard-gradle:6.0.3
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-stdlib:1.3.72.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > com.android.tools.build.jetifier:jetifier-core:1.0.0-beta09
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta09
      > Skipped due to earlier error
   > Could not resolve org.checkerframework:checker-qual:2.11.1.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > org.tensorflow:tensorflow-lite-metadata:0.1.0-rc1
      > Skipped due to earlier error
   > Could not resolve org.checkerframework:checker-qual:2.11.1.
     Required by:
         project :flutter_logs > org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20 > com.google.guava:guava:29.0-jre
      > Skipped due to earlier error
   > Could not resolve org.apache.commons:commons-compress:1.12.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdklib:27.1.3
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdklib:27.1.3 > com.android.tools:repository:27.1.3
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.trove4j:trove4j:20160824.
     Required by:
         project :flutter_logs > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdk-common:27.1.3
      > Skipped due to earlier error
> Failed to notify project evaluation listener.
   > Could not get unknown property 'android' for project ':flutter_logs' of type org.gradle.api.Project.
   > Could not get unknown property 'android' for project ':flutter_logs' of type org.gradle.api.Project.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org/
BUILD FAILED in 4m 30s
```

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -2,9 +2,9 @@ group 'com.flutter.logs.plogs.flutter_logs'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.5.20'
+    ext.kotlin_version = '1.5.31'
     repositories {
-        jcenter()
+        mavenCentral()
         google()
     }
 
@@ -17,7 +17,7 @@ buildscript {
 rootProject.allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
         maven { url 'https://jitpack.io' }
     }
 }

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -9,7 +9,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
+        classpath 'com.android.tools.build:gradle:7.1.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }

--- a/android/gradle/wrapper/gradle-wrapper.properties
+++ b/android/gradle/wrapper/gradle-wrapper.properties
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip

--- a/example/android/build.gradle
+++ b/example/android/build.gradle
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
+        classpath 'com.android.tools.build:gradle:7.1.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }

--- a/example/android/build.gradle
+++ b/example/android/build.gradle
@@ -1,7 +1,7 @@
 buildscript {
-    ext.kotlin_version = '1.5.20'
+    ext.kotlin_version = '1.5.31'
     repositories {
-        jcenter()
+        mavenCentral()
         google()
     }
 
@@ -13,7 +13,7 @@ buildscript {
 
 allprojects {
     repositories {
-        jcenter()
+        mavenCentral()
         google()
         maven { url 'https://jitpack.io' }
     }

--- a/example/android/gradle/wrapper/gradle-wrapper.properties
+++ b/example/android/gradle/wrapper/gradle-wrapper.properties
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip

